### ISSUE TITLE
Submitting R package assignment

### ISSUE THREAD

**Issue by Johaning** [Watts-College/cpp-527-fall-2021#28]
The instructions for the R package assignment says "To receive credit for the assignment, submit the URL to your package on GitHub through Canvas."

I pasted my link into the comment box, but Canvas will not let me submit without attaching a document. Is a Word doc with the link sufficient?

**Comment by mtwelker** [comment]
That's how I submitted it (in a Word document), so I hope that's okay!

**Comment by lecy** [comment]
Yes, that’s fine. There’s no easy way. If you were able to submit in any format I can figure it out.
